src/script/components/Conversation/ConversationCells/useGetAllCellsFiles/useGetAllCellsFiles.ts

```diff
@@ -34,7 +34,9 @@ interface UseGetAllCellsFilesProps {
 }
 
 export const useGetAllCellsFiles = ({cellsRepository, conversationQualifiedId}: UseGetAllCellsFilesProps) => {
-  const {setFiles, setStatus, setError} = useCellsStore();
+  const {setFiles, pageSize, setStatus, setPagination, setError} = useCellsStore();
+
+  const [offset, setOffset] = useState<number>(0);
```

Comment:
there's no need to set the type here, TypeScript will infer it

```suggestion
  const [offset, setOffset] = useState(0);
```

Reply:
👍